src/app/components/core/user-management/user-management.component.html

```diff
@@ -125,7 +145,8 @@ <h6 class="is-6 has-text-weight-semibold">
     </div>
   </div>
   <app-alert-dialog (enabledService)='enableUser($event)' (deleteUserService)='deleteUser($event)'
-    (logoutAllUserSessionsService)='clearAllSessions($event)' [childData]='childData'></app-alert-dialog>
+    (unblockUserService)="unblockUser($event)" (logoutAllUserSessionsService)='clearAllSessions($event)'
+    [childData]='childData'></app-alert-dialog>
```

Comment:
I believe we should use only one variable instead of using 4 different variables i.e. `enabledService, deleteUserService, logoutAllUserSessionsService, unblockUserService` as all of these emitting the same i.e. userID from alert-dialog component. And based on the value of chidData.key, we can call the respected functions.